M2/Macaulay2/packages/ToricHigherDirectImages.m2

```diff
@@ -218,6 +218,13 @@ affineSemigroupGenerators (NormalToricVariety, List) := Matrix => (X, w) -> (
 	)
     )
 
+-- Internal method. computes the "generating lattice points" in a polyhedron via Hilbert bases.
+idealFromPolyhedron = method();
+idealFromPolyhedron Polyhedron := List => P -> (
```

Comment:
If the name includes "ideal", I think the output should also include an ideal, no?